packages/circuit-ui/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -49,6 +51,14 @@ const CheckboxWithState = ({
   );
 };
 
+const CheckboxContainer = styled.div`
+  width: 100%;
+  height: 100px;
+  display: flex;
+  justify-content: center;
+  margin-top: 10%;
+`;
```

Comment:
It looks like you're trying to center the story. Could the [`layout` option](https://storybook.js.org/docs/react/configure/story-layout) help here?